Core/GDCore/Events/Builtin/CommentEvent.cpp

```diff
@@ -12,6 +12,29 @@ using namespace std;
 
 namespace gd {
 
+vector<const gd::String*> CommentEvent::GetAllSearchableStrings( BaseEvent &event)
+    const {
+  vector<const gd::String*> allSearchableStrings;
+
+  stringCom1 = event.GetChild("comment", 0, "Com1").GetValue().GetString();

```

Comment:
You don't need to do this, look at what is already available in CommentEvent:

https://github.com/4ian/GDevelop/blob/7b30f63cb126013417535ba77914cd264b548f19/Core/GDCore/Events/Builtin/CommentEvent.h#L60-L61

Exactly what you need:

```
allSearchableStrings.push_back(com1);
allSearchableStrings.push_back(com2);
```

No need to push a pointer, it's fine here to put a copy of the strings.